test/framework/src/main/java/org/elasticsearch/index/mapper/MapperTestCase.java

```diff
@@ -364,8 +364,18 @@ public final void testEmptyName() {
             minimalMapping(b);
             b.endObject();
         })));
-        assertThat(e.getMessage(), containsString("name cannot be empty string"));
-        assertParseMinimalWarnings();
+        assertThat(e.getMessage(), containsString("field name cannot be an empty string"));
```

Comment:
I think you'll still need the `assertParseMinimalWarnings()` here for the legacy geo mappers?

Reply:
we throw error earlier now, hence we don't get to the deprecations before throwing the empty/blank field name error. the default assert no warnings in the after method is good enough then.